test/e2e/operator_test.go

```diff
@@ -0,0 +1,77 @@
+package e2e_test
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	ingressv1alpha1 "github.com/openshift/cluster-ingress-operator/pkg/apis/ingress/v1alpha1"
+	osv1 "github.com/openshift/cluster-version-operator/pkg/apis/operatorstatus.openshift.io/v1"
+
+	"github.com/operator-framework/operator-sdk/pkg/sdk"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+func TestClusterOperatorStatusNotFailingNotProgressing(t *testing.T) {
```

Comment:
I thought this test would just wait for the operator to publish `OperatorAvailable=ConditionTrue`... not sure why the intermediate conditions are important here?

Reply:
Renamed and simplified to check only the "Available" condition.